lib/src/dependency.dart

```diff
@@ -2,24 +2,25 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:collection/collection.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:pub_semver/pub_semver.dart';
 import 'package:yaml/yaml.dart';
 
 part 'dependency.g.dart';
 
-Map<String, Dependency> parseDeps(Map source) =>
+Map<String, Dependency> parseDeps(Map? source) =>
```

Comment:
Is taking a nullable useful here?

Reply:
Yes! It's used in a converter where `null` might flow throw.